### PR TITLE
Updates to the "Managing cluster alerts" section

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

File: modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc [hunk 1]
```diff
@@ -15,26 +15,18 @@ You can find an alert and see information about it or its governing alerting rul
 
 . Optional: Filter the alerts by state using one or more of the state buttons *Firing*, *Silenced*, *Pending*, *Not firing*.
```
Comment (kyoto) [inline modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc:16]: The `Not Firing` filter has been removed

File: modules/monitoring-getting-information-about-silences.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ You can find a silence and view its details.
 
 . Optional: Filter the silences by state using one or more of the state buttons *Active*, *Pending*, *Expired*.
 
-. Optional: Sort the silences by clicking one or more of the *Name*, *State*, and *Firing alerts* column headers.
+. Optional: Sort the silences by clicking one or more of the *Name*, *Firing alerts*, *State*, and *Creator* column headers.
```
Comment (kyoto) [inline modules/monitoring-getting-information-about-silences.adoc:18]: I guess this should be `Firing Alerts` to match the capitalization in the screenshot

File: modules/monitoring-contents-of-the-alerting-ui.adoc [hunk 3]
```diff
@@ -7,43 +7,43 @@
...
-. Upload a file with Alertmanager configuration.
-. Examine and edit the current Alertmanager configuration.
-. Save the updated Alertmanager configuration.
+. Filtering Alerting Rules by their name.
+. Filtering Alerting Rules by their states.
+. Alerting Rule name.
+. Value of the Severity label of the Alerting Rule.
+. State of the Alert defined by the Alerting Rule.
 
 Also, next to the title of each of these pages is a link to the old Alertmanager interface.
```
Comment (kyoto) [inline modules/monitoring-contents-of-the-alerting-ui.adoc:49]: I don't think we should say `old` here. This is a link to Alertmanager's own UI. Maybe just `link to the Alertmanager interface`?

**Comment by kyoto** [comment]
LGTM. Just a couple of small suggestions. Thanks @rh-max!

**Comment by rh-max** [comment]
@kyoto Thanks! Made changes exactly per your feedback.

**Comment by lihongyan1** [comment]
LGTM

File: modules/monitoring-contents-of-the-alerting-ui.adoc [hunk 4]
```diff
@@ -7,43 +7,43 @@
...
 
-The *Silences* page is accessible by clicking *Monitoring* -> *Alerting* -> *Silences* in the {product-title} web console.
+The *Silences* page:
 
 image::monitoring-silences-screen.png[]
 
 . Creating a silence for an alert.
 . Filtering silences by their name.
 . Filtering silences by their states. If a silence is pending, it is currently not active because it is scheduled to start at a later time. If a silence expired, it is no longer active because it has reached its end time.
 . Description of a silence. It includes the specification of alerts that it matches.
```
Comment (juzhao) [inline modules/monitoring-contents-of-the-alerting-ui.adoc:33]: it is better change to
`Description of a silence. It includes the silence name and the specification of alerts that it matches`

File: modules/monitoring-contents-of-the-alerting-ui.adoc [hunk 5]
```diff
@@ -7,43 +7,43 @@
...
-. Current state of the silence. For active silences, it shows when it ends, and for pending silences, it shows when it starts.
 . Number of alerts that are being silenced by the silence.
+. Current state of the silence. For active silences, it shows when it ends, and for pending silences, it shows when it starts.
```
Comment (juzhao) [inline modules/monitoring-contents-of-the-alerting-ui.adoc:35]: `For active silences, it shows when it ends, and for pending silences, it shows when it starts`
it is hard to understand it

File: modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc [hunk 6]
```diff
@@ -13,28 +13,20 @@ You can find an alert and see information about it or its governing alerting rul
...
-To see alert details, click on the name of the alert. This is the page with alert details:
-+
-image::monitoring-alert-overview.png[]
-+
-The page has the graph with timeseries of the alert. It also has information about the alert, including:
+To see alert details, click on the name of the alert. The page that opens has the graph with timeseries of the alert. It also has information about the alert, including:
 +
 --
 * A link to its governing alerting rule
 * Description of the alert
```
Comment (lihongyan1) [inline modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc:26]: Labels of alert should be added

**Comment by lihongyan1** [comment]
LGTM

**Comment by sichvoge** [comment]
@maximsvistunov can we mention that this is only accessible via the administrator perspective? couldn't find that. In 4.6, we will have something else for the developer perspective.

**Review by lamek — COMMENTED** [review]
Two quick comments. LGTM.

**Comment by rh-max** [comment]
@kyoto @juzhao @lihongyan1 @sichvoge @lamek  Thanks for the reviews. Kevin, can you please merge & cherry-pick into `enterprise-4.5`?

**Comment by rh-max** [comment]
Thanks @lamek .

**Pull request merged**